src/snowflake/snowpark/_internal/data_source/drivers/__init__.py

```diff
@@ -15,3 +16,6 @@
 )
 from snowflake.snowpark._internal.data_source.drivers.sqlite_driver import SqliteDriver
 from snowflake.snowpark._internal.data_source.drivers.pyodbc_driver import PyodbcDriver
+from snowflake.snowpark._internal.data_source.drivers.databricks_driver import (
+    DatabricksDriver,
+)
```

Comment:
nint: i think you can remove the () since there is only one import

Reply:
the git commit hook auto wrapped this for me